io.qt.qtwebengine.BaseApp.json

```diff
@@ -23,7 +23,7 @@
             "config-opts": [
                 "-DQT_FEATURE_webengine_kerberos=ON",
                 "-DQT_FEATURE_webengine_proprietary_codecs=ON",
-                "-DQT_FEATURE_webengine_system_ffmpeg=ON",
+                "-DQT_FEATURE_webengine_system_ffmpeg=OFF",
```

Comment:
why off?

Reply:
The ffmpeg version of the runtime is not compatible with Chromium

Reply:
and it is in the 6.4 branch? That's odd, what's the error?

Reply:
This was explained [here](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1611). There are 3 possible solutions:

1. runtime patches ffmpeg
2. baseapp patches webengine
3. internal ffmpeg is used

Reply:
The thing is that 6.4 and 6.5 use the same freedesktop runtime. So if it works in 6.4 then it works in 6.5

Which is either 6.4 is broken or this should just be the same as 6.4

You can't expect that change in freedesktop runtime as 6.5 will not use a more recent runtime.

Reply:
May also be that 6.5 requires a newer ffmpeg.

Anyhow i think "internal ffmpeg is used" is good enough, if -DQT_FEATURE_webengine_system_ffmpeg=OFF does that no opposition from my side.

Reply:
> So if it works in 6.4 then it works in 6.5

The webengine commit which blocks ffmpeg >= 5 (the problem is runtime ffmpeg is too new not too old) was introduced in [6.5.x](https://invent.kde.org/qt/qt/qtwebengine/-/commit/a09ebdd1a722643384b40bdc177a60ee74a30390#) branch. I don't know if 6.4 is silently broken or not affected.

Reply:
As Erik555 said, it doesn't build with `-DQT_FEATURE_webengine_system_ffmpeg=ON`, so we need to disable that.

It is also highly possible, that this is also broken in the 6.4 branch. I assume they got a bug report, that it is not working, so they added this check.